server/utils/applicationUtils.test.ts

```diff
@@ -32,4 +37,71 @@ describe('applicationUtils', () => {
       )
     })
   })
+
+  describe('getService', () => {
+    beforeAll(() => {
+      config.approvedPremisesRootPath = 'approved-premises-root-path'
+      config.temporaryAccommodationSubdomain = 'approved-premises-root-path'
```

Comment:
This line could possibly be removed in favour of the same option set a few lines below: 

```
config.temporaryAccommodationSubdomain = 'temporary-accommodation-subdomain'
```

Which looks to have the right value assigned to it too.

Reply:
Ah, this should be

```
config.temporaryAccommodation*RootPath* = 'temporary-accommodation-root-path'
```

Will fix